eip155/caip10.md

```diff
@@ -29,6 +33,17 @@ ratification of [EIP55][], however, a particular normalization of lowercase- and
 uppercase- `abcdefABCDEF` characters was invented as an efficient form of
 checksum. See [EIP55][] for specification.
 
+The chain ID will be used to represent blockchain except special case of 0 as chainID to represent EOA.
+
+The `chain_id` string will be ammended as follows:
+
+```
+chain_id:    namespace + ":" + network_id + ":" + reference
+network_id:    [0-9]{1,1}
```

Comment:
```suggestion
network_id:    [0-9]{1,19}
```
based on:
https://ethereum-magicians.org/t/eip-2294-explicit-bound-to-chain-id/11090/24
and
https://github.com/ethereum/EIPs/issues/2294
it seems chainId could be a bigint?